tests/Integration/tests/configuration/input/guides.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<guides xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+        links-are-relative="true"
+        default-code-language="php"
+>
+    <project title="My Project" version="3.1.4" />
```

Comment:
The XSD lists `<extension>` however that one seems a bit dangerous when exposing to be interpreted by foreign extensions, what do you think?

Also a `<theme extends="">` tag exists (don't understand how it relates to the `<guides theme="">` attribute)?

Last a `<template>` tag is mentioned in the XSD (with attributes file, node, format) - is that something we want to expose also to documentation modules?

Reply:
Only extensions that are already loaded via composer can be included with the <extension> tag. So while i would not suggest anyone to tinker with that it is impossible to include an extension that is not already loaded. You could theoretically exclude extensions however -  like if you don't need plant-uml you could exclude that extension to be faster or whatever.

We could, of course also define our own configuration format just for the theme later on